internal/gonut/cmd/push.go

```diff
@@ -259,6 +282,12 @@ func runSampleAppPush(app sampleApp) error {
 
 		bufPrintf("     DimGray{_stack:_} DarkSeaGreen{%s}\n", report.Stack())
 		bufPrintf(" DimGray{_buildpack:_} DarkSeaGreen{%s}\n", report.Buildpack())
+		// set color according to the statuscode
+		if report.StatusCode == 200 {
```

Comment:
Just a minor thing: It helps with the readability to use `http.StatusOK` instead of `200`.